opm/material/fluidsystems/blackoilpvt/BrineCo2Pvt.cpp

```diff
@@ -94,7 +97,9 @@ initFromState(const EclipseState& eclState, const Schedule&)
     // set the surface conditions using the STCOND keyword
     Scalar T_ref = eclState.getTableManager().stCond().temperature;
     Scalar P_ref = eclState.getTableManager().stCond().pressure;
-    
+
+    co2Tables_ = Params();
```

Comment:
This is double work, as `co2Tables_` should already be default-initialized in the constructor.

Reply:
fixed now